app/controllers/step_by_step_pages_controller.rb

```diff
@@ -64,9 +64,8 @@ def publish
     if request.post?
       @publish_intent = PublishIntent.new(params)
       if @publish_intent.valid?
+        note_description = publish_note_description(@publish_intent)
```

Comment:
As an instance variable `@publish_intent` is accessible by the entire object, so it doesn't need to be passed to `publish_note_description`.